xtask/tests/tidy-tests/main.rs

```diff
@@ -20,7 +20,16 @@ fn rust_files_are_tidy() {
 }
 
 fn check_todo(path: &Path, text: &str) {
-    if path.ends_with("tests/cli.rs") {
+    let whitelist = &[
```

Comment:
Not particularly happy about this approach, but this isn't a particularly sophisticated check that would allow more fine grained configuration.

Might be reasonable to extend this to detect some marker in the file that allows more fine-grained ignoring, but idk.

Any suggestions?

Reply:
I think such whitelisting is the optimal solution, given the scale of the problem actually!